flexmeasures/data/schemas/forecasting/pipeline.py

```diff
@@ -66,7 +66,12 @@ def validate_parameters(self, data: dict, **kwargs):
                     "start-predict-date must be before end-date",
                     field_name="start_predict_date",
                 )
-
+        elif not start_date:
+            raise ValidationError(
+                "Missing required option: either --start-date/--train-start or --from-date/--predict-start must be specified. "
```

Comment:
Just lose the `--`s with an eye on using this schema outside of the CLI in the future.